### DISCUSSION TITLE
Linking Audio in Render Notes

### DISCUSSION THREAD

**Discussion by MrOctocat** [TriliumNext/Trilium#6233] [Q&A] [answered]
> I think you missed the last part of the url. It should be like this:
```
# Image type note
api/images/NOTE_ID/ANY_STRING
# Attachment
api/attachments/NOTE_ID/image/ANY_STRING
```

_Originally posted by @Nriver in https://github.com/zadam/trilium/discussions/4763#discussioncomment-9362151_

For MP3 audio attachments, should I do the following?

```
<audio controls>
  <source src="api/attachments/NOTE_ID/audio/ANY_STRING" type="audio/mpeg" />
</audio>
```

Because it doesn't seem to be working.

**Comment by eliandoran** [comment, marked as answer]
Best approach is not to use attachments but a note of type File (simply drag and drop).

Here's a snippet that works with videos, it should work similarly with audio:

```html
<video class="video-preview" controls="" style="width: 988.167px;" src="api/notes/KYuEy2GM99lf/open-partial" type="video/mp4"></video>
```
